src/utils/SpoonacularRequests.ts

```diff
@@ -16,6 +13,21 @@ export const fetchData = async() => {
     }
     return searchString;
   }
-  
+  const call: any = await fetch(`https://api.spoonacular.com/recipes/findByIngredients?apiKey=ec8da5b51af4425698dd91381535afff&ingredients=${ingredientSearch(ingredientList)}`);
+
+  const response = await call.json()
+
+  const onCallHandler = () => {
```

Comment:
could you separate this out of this function -- make it another util and import it